src/entities/trades/utils.ts

```diff
@@ -60,7 +60,7 @@ export const RPC_PROVIDER_LIST: Record<ChainId, string> = {
   [ChainId.ARBITRUM_GOERLI]: 'https://goerli-rollup.arbitrum.io/rpc',
   [ChainId.POLYGON]: 'https://polygon-rpc.com',
   [ChainId.GOERLI]: 'https://goerli.infura.io/v3/e1a3bfc40093494ca4f36b286ab36f2d',
-  [ChainId.OPTIMISM_MAINNET]: 'https://mainnet.optimism.io',
+  [ChainId.OPTIMISM_MAINNET]: 'https://opt-mainnet.g.alchemy.com/v2/6cRVjVO2uOTC9gWFCsBnquUwOM9zuWQZ',
```

Comment:
What's wrong with the public RPC?

Reply:
it was too slow....I was getting random errors as well because of how slow it is and couldn't even test it out properly...